packages/service-library/src/servicelib/fastapi/long_running_tasks/_routes.py

```diff
@@ -41,7 +41,7 @@ async def get_task_result(
 ) -> TaskResult:
     assert request  # nosec
 
-    task_result = task_manager.get_task_result(task_id=task_id)
+    task_result = task_manager.get_task_result_old(task_id=task_id)
```

Comment:
shouldn't you add a TODO with an issue number here to update to the analogous interface when possible?

Reply:
yes